packages/apollo-datasource-rest/src/RESTDataSource.ts

```diff
@@ -258,7 +259,7 @@ export abstract class RESTDataSource<TContext = any> extends DataSource {
             cacheKey,
             cacheOptions,
           });
-          return await this.didReceiveResponse(response, request);
+          return await this.didReceiveResponse(response, request, this.context);
```

Comment:
Hmm, feel free to back out the changes to `RESTDataSource` if this is going too far, but I didn't realize that `this.context` was already on the instance.  Therefore, passing it into `didReceiveResponse` seems seem unnecessary since it can just be accessed from the `didReceiveResponse` method as `this.context` already!

Reply:
Yeah, agreed.
fa80407